stb_dxt.h

```diff
@@ -651,23 +664,15 @@ static void stb__InitDXT()
 
 void stb_compress_dxt_block(unsigned char *dest, const unsigned char *src, int alpha, int mode)
 {
-   unsigned char data[16][4];
    static int init=1;
    if (init) {
       stb__InitDXT();
       init=0;
    }
 
    if (alpha) {
-      int i;
       stb__CompressAlphaBlock(dest,(unsigned char*) src+3, 4);
       dest += 8;
-      // make a new copy of the data in which alpha is opaque,
```

Comment:
Why is this code block deleted? According to the comment, the color code expects the alpha to be opaque for the color code's constancy test to work. Fixing the alpha-constant case doesn't seem to help this.

Reply:
See change above on line #521 Instead of copying color data without alpha to tmp buf, the check in the loop is modified to remove alpha from the test when checking equality.

Reply:
Oh right, I missed that. Continued discussion moved to main thread.